_layouts/default.html

```diff
@@ -9,6 +9,7 @@
 		<meta name="language" content="de"/>
 		<meta name="content-language" content="de"/>
 		<meta name="publisher" content="simplabs, Malthe Luda" />
+    <meta name="apple-itunes-app" content="app-id=918237891, affiliate-data=myAffiliateData, app-argument=myURL">
```

Comment:
Can we leave out `affiliate-data` and `app-argument` as we only provide dummy data anyway?